setup.cfg

```diff
@@ -23,8 +23,9 @@ exclude = jwst/extern,docs,jwst/associations,jwst/jwpsf,jwst/ramp_fitting, jwst/
 [tool:pytest]
 minversion = 3
 addopts = --ignore=build
-norecursedirs = .eggs build docs/_build relic
+norecursedirs = .eggs build docs/_build relic jwst/timeconversion jwst/lib jwst/extern jwst/pipeline/tests
```

Comment:
Is there a reason `jwst/pipeline/tests` is added here?  There's actually a bunch of tests in there that are run using `pytest`.